frontend/model/contracts/chatroom.js

```diff
@@ -695,7 +693,8 @@ sbp('chelonia/defineContract', {
   methods: {
     'gi.contracts/chatroom/_cleanup': ({ contractID, state }) => {
       if (state?.members) {
-        sbp('chelonia/contract/release', Object.keys(state.members)).catch(e => {
+        const activeMemberIds = Object.keys(state.members).filter(memberID => !state.members[memberID].hasLeft)
```

Comment:
Is this another one that should be replaced with the new getter?

Reply:
Possibly. Currently, `_cleanup` doesn't get passed `getters`, which is why I'm not using it.